plugins/net_plugin/net_plugin.cpp

```diff
@@ -122,16 +123,16 @@ namespace eos {
         sync_received(),
         sync_requested(),
         socket( s ),
-        pending_message_size(),
-        pending_message_buffer(),
+        pending_message_size(0),
+        pending_message_buffer(1024*1024*4),
+        send_buffer(1024*1024*4),
         remote_node_id(),
         last_handshake(),
         out_queue(),
         connecting (false),
-        syncing (false),
         peer_addr ()
     {
-      wlog( "created connection from client" );
+      wlog( "created network connection" );
```

Comment:
Why did you change this? I chose "connection from client" specifically since that's what it is. I guess with a minute or two of thought we could combine these two constructors and pass a smarter value for the peer_addr initializer to indicate the role of this connection.

Reply:
At the moment, that isn't always what it is.  In my testing using `launcher`, simply connecting two `eosd` peers consistently used the constructor that output `created connection from client`.  Unless you're choosing to use language that makes no distinction between peer `eosd`s and clients like `eosc` in the description, considering them all clients, my change more accurately describes the code's present lack of distinguishing between them.